WSL/basic-commands.md

```diff
@@ -168,8 +168,44 @@ To terminate the specified distribution, or stop it from running, replace `<Dist
 
 ## Identify IP address
 
-- `wsl hostname -i` for the IP address of your Linux distribution installed via WSL 2 (the WSL 2 VM address)
```

Comment:
This should be kept simple as just a quick two line explanation, and we should link out deeper info to the networking page.